lib/opentok/archives.rb

```diff
@@ -146,5 +152,16 @@ def delete_by_id(archive_id)
       (200..300).include? response.code
     end
 
+    def layout(archive_id, options = {})
```

Comment:
There are no docs comments for this method. (You can write them based on the `Archive.layout(options)` method.